docs/en/StaticRoutes.md

```diff
@@ -0,0 +1,105 @@
+# Static Gateway Endpoints
+
+Most payment providers accept dynamic callback URLs where they send status-updates to. 
+A dynamic callback URL looks like this: `https://example.com/paymentendpoint/c62f90dc62bdbf4966b58e38d6f767/complete`.
+
+It contains the payment identifier and action and will be sent as callback url with the payment request. 
+This URL is only valid for one payment. Using this kind of callback is the default behavior and you should not use anything else, if not forced to!
+
+In some rare cases, payment providers *only* allow a single callback URL (which is normally being set in the admin interface of the payment provider).
+This callback url will receive all responses from that given provider and must therefore be capable of finding and processing the appropriate payment with the given request payload.
+An example of such a static callback URL: `https://example.com/paymentendpoint/BarclaysEpdq_Essential/complete`.
+
+In order to make static-routes work for your payment-provider, you need to do the following:
+
+### Enable the static route in your config file
+
+Use the `use_static_route` flag to enable the static route for your gateway. Only enable this for the gateway(s) that actually *need* a static route.
+
+Example:
+
+```yaml
+GatewayInfo:
+  BarclaysEpdq_Essential:
+    use_static_route: true
+    parameters:
+      clientId: xxxxxx
+      shaIn: abcdefghijk
+```
+
+### Write an Extension that gets a payment from the incoming request data
+
+With static routes we have no payment id to easily look up the payment. You have to implement the lookup yourself. 
+*If you implement such an extension for a payment provider, please contribute it back to us via pull-request*!
+
+Your extension should implement the following method:
+
+`updatePaymentFromRequest(SS_HTTPRequest $request, $gateway) : Payment`
+
+The `$request` parameter is the incoming HTTP request. You can use this to look up the Payment with the incoming request data.
+The `$gateway` parameter is a string and will contain the name of the gateway. Your method **must** check if the gateway name matches and only return a payment for the gateway your extension is responsible for!
+
```

Comment:
It might be worth mentioning that in this example I passed the orderID param to Barclays myself and then listened for it's return.

This could easily be the payment id however if you were just using this module standalone (although you would still have to listen for it, retrieve it and get a payment.